great_expectations/analytics/events.py

```diff
@@ -0,0 +1,13 @@
+from typing import ClassVar, List
+
+from great_expectations.analytics.actions import DATA_CONTEXT_INITIALIZED
+from great_expectations.analytics.base_event import Action, Event
+
+
+class DataContextInitializedEvent(Event):
+    action = DATA_CONTEXT_INITIALIZED
+
+    _allowed_actions: ClassVar[List[Action]] = [DATA_CONTEXT_INITIALIZED]
+
+    def __init__(self):
+        super().__init__(action=self.action)
```

Comment:
The flow here feels off to me. This and the parent class both define an `action` field, and we read this during init and pass it to the parent, which under the hood assigns it to self.action again. I would remove line 8 and just call `super().__init__(action=DATA_CONTEXT_INITIALIZED)`